docs/news.rst

```diff
@@ -3,8 +3,430 @@
 Release notes
 =============
 
-.. note:: Scrapy 1.x will be the last series supporting Python 2. Scrapy 2.0,
-          planned for Q4 2019 or Q1 2020, will support **Python 3 only**.
+.. _release-2.0.0:
+
+Scrapy 2.0.0 (2020-02-2?)
+-------------------------
+
+Highlights:
+
+* Python 2 support has been removed
+* :doc:`Partial <topics/coroutines>` :ref:`coroutine syntax <async>` support
+  and :doc:`experimental <topics/asyncio>` :mod:`asyncio` support
+* New :meth:`Response.follow_all <scrapy.http.Response.follow_all>` method
+* :ref:`FTP support <media-pipeline-ftp>` for media pipelines
+* IPv6 support through :setting:`DNS_RESOLVER`
+
+Backward-incompatible changes
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+*   Python 2 support has been removed, following `Python 2 end-of-life on
+    January 1, 2020`_ (:issue:`4091`, :issue:`4114`, :issue:`4115`,
+    :issue:`4121`, :issue:`4138`, :issue:`4231`, :issue:`4242`, :issue:`4304`,
+    :issue:`4309`, :issue:`4373`)
+
+*   Retry gaveups (see :setting:`RETRY_TIMES`) are now logged as errors instead
+    of as debug information (:issue:`3171`, :issue:`3566`)
+
+*   File extensions that
+    :class:`LinkExtractor <scrapy.linkextractors.lxmlhtml.LxmlLinkExtractor>`
+    ignores by default now also include ``7z``, ``7zip``, ``apk``, ``bz2``,
+    ``cdr``, ``dmg``, ``ico``, ``iso``, ``tar``, ``tar.gz``, ``webm``, and
+    ``xz`` (:issue:`1837`, :issue:`2067`, :issue:`4066`)
+
+*   The :setting:`METAREFRESH_IGNORE_TAGS` setting is now an empty list by
+    default, following web browser behavior (:issue:`3844`, :issue:`4311`)
+
+*   The
+    :class:`~scrapy.downloadermiddlewares.httpcompression.HttpCompressionMiddleware`
+    now includes spaces after commas in the value of the ``Accept-Encoding``
+    header that it sets, following web browser behavior (:issue:`4293`)
+
+*   The ``__init__`` method of download handlers (see
+    :setting:`DOWNLOAD_HANDLERS`) no longer receives a ``settings`` parameter;
+    use the ``from_settings`` or ``from_crawler`` class methods to expose such
+    a parameter to your custom download handlers (:issue:`4126`)
+
+*   Overridden settings are now logged in a different format. This is more in
+    line with similar information logged at startup (:issue:`4199`)
+
+.. _Python 2 end-of-life on January 1, 2020: https://www.python.org/doc/sunset-python-2/
+
+Changes to scheduler queue classes
```

Comment:
What do you think about moving all the details somewhere lower, maybe to a separate section, to highlight new features / deprecations / etc. more?

Reply:
Also, this can be in highlights, if we find a succinct way to explain the change

Reply:
Could you elaborate?

We could sort “highlights → new features → bug fixes → deprecation removals → deprecations → backward-incompatible changes → documentation → QA”. Do you have something else in mind?

Reply:
I was thinking that we can say that there is a backwards incompatible change in Scheduler/queues in this section, but have a separate section at the end of the release notes, with a more detailed description. The goal is to show new features & bug fixes sooner, to get people more excited about the upgrade - backwards incompatible changs mean more work for  people, while new features & fixes mean less work. But I'm fine with merging as-is. This is nitpicking, and is super-minor.

Reply:
Moved to a section at the referenciable section at the end ([rendering](https://scrapy-gallaecio.readthedocs.io/en/release-notes/news.html#scheduler-queue-changes)).